applications/IgaApplication/python_scripts/iga_output_process.py

```diff
@@ -0,0 +1,151 @@
+                for node in self.model_part.Nodes:
+                    value = node.GetSolutionStepValue(variable, 0)
+                    output_file.write(str(node.Id) + "  " + str(value) + "\n")
+                output_file.write("End Values\n")
+
+            for variable in self.nodal_results_vector:
+                output_file.write("Result \"" + variable.Name() + "\" \"Load Case\" " + str(label) + " Vector OnNodes\nValues\n")
+                for node in self.model_part.Nodes:
+                    value = node.GetSolutionStepValue(variable, 0)
+                    output_file.write(str(node.Id) + "  " + str(value[0]) + "  " +  str(value[1]) + "  " + str(value[2]) + "\n")
+                output_file.write("End Values\n")
+
+            for variable in self.integration_point_results_scalar:
+                output_file.write("Result \"" + variable.Name() + "\" \"Load Case\" " + str(label) + " Scalar OnGaussPoints\nValues\n")
+                for element in self.model_part.Elements:
+                    value = element.CalculateOnIntegrationPoints(variable, self.model_part.ProcessInfo)[0]
+                    output_file.write(str(element.Id) + "  " + str(value) + "\n")
+                output_file.write("End Values\n")
+
+            for variable in self.integration_point_results_vector:
+                output_file.write("Result \"" + variable.Name() + "\" \"Load Case\" " + str(label) + " Vector OnGaussPoints\nValues\n")
+                for element in self.model_part.Elements:
+                    value = element.CalculateOnIntegrationPoints(variable, self.model_part.ProcessInfo)[0]
+                    output_file.write(str(element.Id) + "  " + str(value[0]) + "  " +  str(value[1]) + "  " + str(value[2]) + "\n")
+                output_file.write("End Values\n")
+
+        # Schedule next output
+        if self.output_frequency > 0.0: # Note: if == 0, we'll just always print
+            if self.output_control_is_time:
+                while GetPrettyTime(self.next_output) <= time:
+                    self.next_output += self.output_frequency
+            else:
+                while self.next_output <= self.step_count:
+                    self.next_output += self.output_frequency
+
+    def IsOutputStep(self):
+        if self.output_control_is_time:
+            time = GetPrettyTime(self.model_part.ProcessInfo[KratosMultiphysics.TIME])
+            return (time >= GetPrettyTime(self.next_output))
+        else:
+            return ( self.step_count >= self.next_output )
+
+def GetPrettyTime(time):
+    pretty_time = "{0:.12g}".format(time)
+    pretty_time = float(pretty_time)
+    return pretty_time
+
+def CreateVariablesListFromInput(param):
+    '''Parse a list of variables from input.'''
+    scalar_variables = []
+    vector_variables = []
+    addmissible_scalar_types = ["Bool", "Integer", "Unsigned Integer", "Double", "Component"]
```

Comment:
```suggestion
    admissible_scalar_types = ["Bool", "Integer", "Unsigned Integer", "Double"]
```
this no longer exists

Reply:
Yoo